src/ray/gcs/gcs_server/gcs_placement_group_scheduler.cc

```diff
@@ -176,13 +57,20 @@ void GcsPlacementGroupScheduler::ScheduleUnplacedBundles(
   RAY_LOG(DEBUG) << "Scheduling placement group " << placement_group->GetName()
                  << ", id: " << placement_group->GetPlacementGroupID()
                  << ", bundles size = " << bundles.size();
-  auto scheduling_result = scheduler_strategies_[strategy]->Schedule(
-      bundles,
-      GetScheduleContext(placement_group->GetPlacementGroupID()),
-      cluster_resource_scheduler_);
 
-  auto result_status = scheduling_result.first;
-  auto selected_nodes = scheduling_result.second;
+  std::vector<const ResourceRequest *> resource_request_list;
```

Comment:
To be clear:
These 5 lines are the function of original `GetRequiredResourcesFromBundles`.
It is too simple and no need to abstract to a method.